source/includes/_troubleshooting.md

```diff
@@ -33,3 +33,20 @@ To resolve this, follow steps at [Requesting Access and Refresh Tokens](https://
     "message": "Unauthorized"
 }
 ```
+
+### 403 - Forbidden
+
+#### HIPAA Permissions
```

Comment:
While you are just following patterns you found on this page, I don't think it looks very good.

What do you think about putting the example responses directly below the text describing the response.  The column to the right is supposed to represent a command line shell where we run some code and see the response.  It doesn't work very well for this paradigm.

I would prefer it looked more like https://reachlocal.github.io/api-docs/#api-access-step-2-authorization-security-considerations.

Reply:
I've updated this to be an inline block instead and I agree, it does look much better